### PR TITLE
alias-pages: add pt_BR translation

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by marchersimon** [comment]
@navarroaxel do you think we should merge this or #6462 first?

File: contributing-guides/alias-pages.md [hunk 1]
```diff
@@ -220,7 +220,15 @@ Not translated yet.
 ---
 ### pt_BR
 
-Not translated yet.
+```markdown
+# example
+
+> Este comando é um alias de 'example'.
+
+- Ver documentação para o comando original:
```
Comment (navarroaxel) [inline contributing-guides/alias-pages.md:228]:
```suggestion
> Este comando e um pseudônimo de 'example'.

- Ver documentação sobre o comando original:
```

**Comment by blueskyson** [comment]
> do you think we should merge this or #6462 first?

Sorry, I merged #6462 before seeing this PR.

File: contributing-guides/translation-templates/alias-pages.md [hunk 2]
```diff
@@ -221,7 +221,15 @@ Not translated yet.
 ---
 ### pt_BR
 
-Not translated yet.
+```markdown
+# example
+
+> Este comando é um alias de 'example'.
```
Comment (marchersimon) [inline contributing-guides/translation-templates/alias-pages.md:227]:
```suggestion
> Este comando é um alias de `example`.
```

**Review by marchersimon — APPROVED** [review]
Thanks

**Review by blueskyson — APPROVED** [review]

**Pull request merged**